tldw_Server_API/app/api/v1/endpoints/sharing.py

```
@@ -65,7 +65,7 @@ def _get_token_service():
 
 def _get_audit_service():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_get_audit_service`

(ANN202)